css/style.css

```diff
@@ -1,743 +1,760 @@
-
-  /* font weights */
-  --main-headings-400: 400;
-  --main-headings-500: 500;
-  --main-headings-600: 600;
-  --main-headings-regular: 700;
-  --main-headings-800: 800;
-  --main-headings-900: 900;
-  --body-text-100: 100;
-  --body-text-200: 200;
-  --body-text-300: 300;
-  --body-text-400: 400;
-  --body-text-regular: 500;
-  --body-text-600: 600;
-
-  /* colors */
-  --bg-color: #fafafa;
-  --secondary-color: #ffffff;
-  --text-color: #1a1a1a;
-  --form-select: #000;
-  --form-unselected: #808080;
-  --form-unselected: rgb(92, 92, 92);
-  --form-gray: #c5c5c5;
-  --hr-color: #8e8e8e;
-  --amaranth: #d94252;
-  --teal: #007a80;
-  --dark-teal: #00585c;
-  --light-teal: #00c6d0;
-  --cybergrape: #6c4681;
-  --onyx: #424241;
-  --yaleblue: #00478f;
-  --hotpink: #fa6cb7;
-  --citrine: #d3c53f;
-  --citron: #92a32d;
-  --bcitron: #b37122;
-
-  /* font sizes */
-  --font50: 0.875rem;
-  --font100: 1rem;
-  --font200: 1.25rem;
-  --font300: 1.5rem;
-  --font400: 1.75rem;
-  --font500: 2rem;
-  --font600: 2.5rem;
-  --font700: 3rem;
-  --font800: 3.5rem;
-  --font900: 4rem;
-  --bold: bold;
-  --underline: underline;
-
-  /* variable for wrapper width */
-  --wrapper-width: 90%;
-
-  /* spacing variables */
-  --spacing-small: 1rem;
-  --spacing-intermediate: 1.5rem;
-  --spacing-medium: 2rem;
-  --spacing-large: 4rem;
-  --spacing-xl: 6rem;
-  --spacing-xxl: 12rem;
-  --spacing-xxxl: 22rem;
+body {
+	background-color: var(--bg-color);
 }
 
 /* GLOBAL STYLES (Desktop First for now) */
 /* global img style */
 img {
-  width: 100%;
-  /* removes white space */
-  display: block;
+	width: 100%;
+	/* removes white space */
+	display: block;
 }
 
 .learn-about-us {
-  color: var(--Black, #1a1a1a);
-  font-family: "Lato", sans-serif;
-  font-size: 24px;
-  font-style: normal;
-  font-weight: 500;
-  line-height: 32px;
-  position: absolute;
-  margin: 19px 0;
+	color: var(--Black, #1a1a1a);
+	font-family: 'Lato', sans-serif;
+	font-size: 24px;
+	font-style: normal;
+	font-weight: 500;
+	line-height: var(--line-height-lg);
+	position: absolute;
+	margin: 19px 0;
 }
 
 h2 {
-  color: var(--Teal, #007a80);
-  font-family: Josefin Sans;
-  font-size: var(--font900);
-  font-style: normal;
-  font-weight: 700;
-  margin-top: var(--spacing-large);
+	color: var(--Teal, #007a80);
+	font-family: Josefin Sans;
+	font-size: var(--font900);
+	font-style: normal;
+	font-weight: 700;
+	margin-top: var(--spacing-large);
 }
 
 /* Theres was small space on the left of the ul so this is to get rid that */
 ul {
-  list-style: none;
-  padding: 0px;
+	list-style: none;
+	padding: 0px;
 }
 
 hr {
-  margin-top: var(--spacing-large);
-  height: 2px;
-  background-color: var(--hr-color);
+	margin-top: var(--spacing-large);
+	height: 2px;
+	background-color: var(--hr-color);
 }
 /* global wrapper class */
 .wrapper {
-  width: var(--wrapper-width);
-  max-width: 1440px;
-  margin: 0 auto;
+	width: var(--wrapper-width);
+	max-width: 1230px;
```

Comment:
Changed this to match the Figma design, discovered it was causing issues with the form layout.

Reply:
didn't notice the width of the figma file - great point!